### DISCUSSION TITLE
Help! I have a shi*** water meter

### DISCUSSION THREAD

**Discussion by busaku** [jomjol/AI-on-the-edge-device#993] [Q&A] [answered]
Hey guys,

i have a shi** water meter because my 0,001 number are without numbers:

<img width="374" alt="image" src="https://user-images.githubusercontent.com/25269085/189424278-d31b0b76-2827-4c03-b999-504140c5cf09.png">

(the blue is just a bad try to put a "0" on the top)

Did i have any chance to get correct values? :(
maybe i have to try to paint the numbers on the glas correctly

**Comment by jomjol** [comment]
The numbers are not the problem. This kind of pointer is not part of the training. You need to add the images with this tool:
https://github.com/haverland/collectmeteranalog

**Reply by busaku** [reply]
i will try it if i have enough log-images :)

**Reply by busaku** [reply]
Is it possible to download the images manually (from SD card?).
WLAN sucks in basement and it takes around 10 minutes for 4 images to download :D

**Comment by ghost** [comment, marked as answer]
I do not think that paint the numbers helps. :-) 
 I would try it. Mostly the real last analog is not needed. You can use the first two after the digitals with an option that extends the second analog to 6.7 in your picture. So you lose only the last analog. If you not want measure a glas of water, it would enough.
 
 And you can help us to improve the neuronal networks with your data like jomol described.

**Reply by busaku** [reply]
How can i set that my last number will be "extended" ? 

Sounds good, but i will try to add my pointer to the neuronal network too :)

Is it possible to "collect" my 0,001 value as images to add it later to the neuronal network but did not count to my value?

**Reply by ghost** [reply]
It's under Post Processing in configuration
<img width="933" alt="image" src="https://user-images.githubusercontent.com/412645/189430837-b88c9a3f-e9b6-4601-b7e9-bdf8dc1e1001.png">

**Reply by ghost** [reply]
To you second question. In this case you need to setup the ROI and log the images a few days. It's detailed described in the readme.md of https://github.com/haverland/collectmeteranalog 

<img width="957" alt="image" src="https://user-images.githubusercontent.com/412645/189431361-4090f382-b10e-4d0d-9a5d-a8684e53594b.png">

**Reply by busaku** [reply]
Thank you :)

Looks good.
I have add an sec. number "collect" only to collect images.
The main-number works nice with extended resolution :)

so, now i wait until i got enough images to train the network :)
